epochStart/metachain/auctionListDisplayer.go

```diff
@@ -9,56 +9,61 @@ import (
 
 	"github.com/multiversx/mx-chain-core-go/core"
 	"github.com/multiversx/mx-chain-core-go/display"
+	"github.com/multiversx/mx-chain-go/config"
 	"github.com/multiversx/mx-chain-go/state"
 	logger "github.com/multiversx/mx-chain-logger-go"
 )
 
 const maxPubKeyDisplayableLen = 20
 const maxNumOfDecimalsToDisplay = 5
 
-func (als *auctionListSelector) displayMinRequiredTopUp(topUp *big.Int, startTopUp *big.Int) {
-	if log.GetLevel() > logger.LogDebug {
-		return
-	}
+type auctionListDisplayer struct {
+	softAuctionConfig *auctionConfig
+}
 
-	if topUp.Cmp(als.softAuctionConfig.minTopUp) > 0 {
-		topUp = big.NewInt(0).Sub(topUp, als.softAuctionConfig.step)
+// NewAuctionListDisplayer creates an auction list data displayer, useful for debugging purposes during selection process
+func NewAuctionListDisplayer(auctionConfig config.SoftAuctionConfig, denomination int) (*auctionListDisplayer, error) {
+	softAuctionConfig, err := getAuctionConfig(auctionConfig, denomination)
+	if err != nil {
+		return nil, err
 	}
 
-	iteratedValues := big.NewInt(0).Sub(topUp, startTopUp)
-	iterations := big.NewInt(0).Div(iteratedValues, als.softAuctionConfig.step).Int64()
-	iterations++
-
-	log.Debug("auctionListSelector: found min required",
-		"topUp", getPrettyValue(topUp, als.softAuctionConfig.denominator),
-		"after num of iterations", iterations,
-	)
+	return &auctionListDisplayer{
+		softAuctionConfig: softAuctionConfig,
+	}, nil
 }
 
-func getShortKey(pubKey []byte) string {
-	pubKeyHex := hex.EncodeToString(pubKey)
-	displayablePubKey := pubKeyHex
-
-	pubKeyLen := len(displayablePubKey)
-	if pubKeyLen > maxPubKeyDisplayableLen {
-		displayablePubKey = pubKeyHex[:maxPubKeyDisplayableLen/2] + "..." + pubKeyHex[pubKeyLen-maxPubKeyDisplayableLen/2:]
+// DisplayOwnersData will display initial owners data for auction selection
+func (ald *auctionListDisplayer) DisplayOwnersData(ownersData map[string]*OwnerAuctionData) {
+	if log.GetLevel() > logger.LogDebug {
+		return
 	}
 
-	return displayablePubKey
-}
-
-func getShortDisplayableBlsKeys(list []state.ValidatorInfoHandler) string {
-	pubKeys := ""
+	tableHeader := []string{
+		"Owner",
+		"Num staked nodes",
+		"Num active nodes",
+		"Num auction nodes",
+		"Total top up",
+		"Top up per node",
+		"Auction list nodes",
+	}
 
-	for idx, validator := range list {
-		pubKeys += getShortKey(validator.GetPublicKey())
-		addDelimiter := idx != len(list)-1
-		if addDelimiter {
-			pubKeys += ", "
+	lines := make([]*display.LineData, 0, len(ownersData))
+	for ownerPubKey, owner := range ownersData {
+		line := []string{
+			hex.EncodeToString([]byte(ownerPubKey)),
```

Comment:
don't we want this bech32?

Reply:
Agree. Please do this properly by injecting 2 address converters on the constructor of the auctionListDiplayer

Reply:
Yes, but was thinking about doing it on another PR.
Will do it here